update: switching to clang for Skia (#352)
JetBrains/kotlin-multiplatform-dev-docs: topics/whats-new/whats-new-compose-180.md

## Desktop

### Software rendering improvement on Windows

Switching to the recommended clang compiler for Skia sped up Windows rendering that relies on the CPU.
This mostly affects pure software rendering (generally rendering relies on the GPU, with only some computations done on the CPU).

> Therefore the improvement is very noticeable on virtual machines like Parallels:

Reviewer: ```suggestion
Therefore the improvement is very noticeable on some virtual machines and on few old GPUs that doesn't support DirectX 12:
```

> ## Desktop
> 
> ### Software rendering improvement on Windows
> 
> Switching to the recommended clang compiler for Skia on Windows sped up rendering that relies on the CPU.
> This mostly affects pure software rendering (generally rendering relies on the GPU, with only some computations done on the CPU).

Reviewer 2: ```suggestion
## Desktop

### Software rendering improvement on Windows

Switching to the recommended clang compiler for Skia on Windows sped up rendering that relies on the CPU.
This mostly affects pure software rendering, as rendering generally relies on the GPU, with only some computations done on the CPU.
```

Therefore the improvement is very noticeable on some virtual machines

> and with a few old graphics cards that [aren't supported by Skia](https://github.com/JetBrains/skiko/blob/30df516c1a1a25237880f3e0fe83e44a13821292/skiko/src/jvmMain/kotlin/org/jetbrains/skiko/GraphicsApi.jvm.kt#L13):

Reviewer 2: ```suggestion
and older graphics cards that [are not supported by Skia](https://github.com/JetBrains/skiko/blob/30df516c1a1a25237880f3e0fe83e44a13821292/skiko/src/jvmMain/kotlin/org/jetbrains/skiko/GraphicsApi.jvm.kt#L13):
```

Windows apps produced by Compose Multiplatform are now up to 6 times as fast in those environments

> (compared to Compose Multiplatform 1.7.3).

Reviewer 2: ```suggestion
compared to Compose Multiplatform 1.7.3.
```

> This, in addition to the Windows for ARM64 support, makes Compose Multiplatform UIs on virtual Windows systems

Reviewer 2: ```suggestion
This improvement, in addition to the Windows for ARM64 support, makes Compose Multiplatform UIs on virtual Windows systems
```

> This, in addition to the Windows for ARM64 support, makes Compose Multiplatform UIs on virtual Windows systems

Reviewer: Worth to mention virtual Windows systems on macOs. Because Virtual machines on Windows/Linux most probably use x64
